src/common/types/metadataQueries.js

```diff
@@ -12,11 +12,15 @@ type MetadataQueryResponseEntry = {
     metadata: MetadataQueryResponseEntryMetadata,
 };
 
-type MetadataQueryResponse = {
+type MetadataQueryResponseData = {
     entries: Array<MetadataQueryResponseEntry>,
     next_marker?: string,
 };
 
+type MetadataQueryResponse = {
```

Comment:
Is this flow type needed? Seems like the only place referenced destructures out `{ data: MetadataQueryResponseData }` already

Reply:
@ConradJChan https://github.com/box/box-ui-elements/pull/1716/files#r344310987